app/views/queries/show.html.erb

```diff
@@ -1,71 +1,86 @@
-<ol class="breadcrumb">
-  <li><%= link_to 'Queries', queries_path %></li>
-  <li class="active"><%= @query.display_name %></li>
-</ol>
+<%= render "govuk_publishing_components/components/breadcrumbs", {
+  breadcrumbs: [
+    {
+      title: "Queries",
+      url: queries_path
+    },
+    {
+      title: "Queries"
+    }
+  ]
+} %>
```

Comment:
It looks like there's some duplication - could this be put into a shared view?

Reply:
new partial introduced